zeppelin-interpreter/src/main/java/org/apache/zeppelin/conf/ZeppelinConfiguration.java

```diff
@@ -1106,6 +1106,7 @@ public enum ConfVars {
 
     // Used by K8s and Docker plugin
     ZEPPELIN_DOCKER_CONTAINER_IMAGE("zeppelin.docker.container.image", "apache/zeppelin:" + Util.getVersion()),
+    ZEPPELIN_DOCKER_CONTAINER_HOME("zeppelin.docker.container.home", "/opt/zeppelin"),
 
     ZEPPELIN_DOCKER_CONTAINER_SPARK_HOME("zeppelin.docker.container.spark.home", "/spark"),
```

Comment:
I also suggest using `/opt/spark` as SPARK_HOME, which matches users who experienced Spark on K8s

Reply:
Then I'll standardize both the Spark installation directory in `scripts/docker/interpreter/Dockerfile` and `ZEPPELIN_DOCKER_CONTAINER_SPARK_HOME` to `/opt/spark`.